frontend/app/player/MessageDistributor/managers/DOM/DOMManager.ts

```diff
@@ -103,6 +103,7 @@ export default class DOMManager extends ListWalker<Message> {
         pNode.sheet && 
         pNode.sheet.cssRules &&
         pNode.sheet.cssRules.length > 0 &&
+        pNode.innerText &&
```

Comment:
whats going on here

Reply:
I had a `cannot call method trim() on undefined` error, and I could find the thing that was undefined, so I added undefined checks to all instances where trim was used.